codersdk/inboxnotification.go

```diff
@@ -10,6 +10,13 @@ import (
 	"github.com/google/uuid"
 )
 
+const (
+	FallbackIconWorkspace = "DEFAULT_WORKSPACE_ICON"
+	FallbackIconAccount   = "DEFAULT_ACCOUNT_ICON"
+	FallbackIconTemplate  = "DEFAULT_TEMPLATE_ICON"
+	FallbackIconOther     = "DEFAULT_OTHER_ICON"
```

Comment:
nit: you may want to change the format to `DEFAULT_ICON_*` so frontend can just check the prefix of the icon string :)